configure.ac

```diff
@@ -174,45 +194,47 @@ AC_ARG_ENABLE([fatal-warnings],
     [AS_HELP_STRING([--enable-fatal-warnings],
         [enable pedantic and fatal warnings for gcc @<:@yes@:>@])],
 )
+yes_no_try "$enable_fatal_warnings" "try"
```

Comment:
Did you mean to change this one to try?

Reply:
Yes, because that was the effective behavior before (anything other than no was treated as try; now yes and try are distinct). But I also meant to update the default in the help so I'll do that